templates/deployment-sidekiq.yaml

```diff
@@ -206,6 +206,24 @@ spec:
             - name: OTEL_SERVICE_NAME_SEPARATOR
               value: "{{ coalesce $context.Values.mastodon.sidekiq.otel.nameSeparator $context.Values.mastodon.otel.nameSeparator }}"
             {{- end }}
+            {{- if $context.Values.mastodon.metrics.prometheus.enabled }}
+            - name: MASTODON_PROMETHEUS_EXPORTER_ENABLED
+              value: "true"
+            - name: PROMETHEUS_EXPORTER_HOST
```

Comment:
Those 2 variables are only needed if `MASTODON_PROMETHEUS_EXPORTER_LOCAL` is not configured

Reply:
Got it, removed